verticapy/performance/vertica/qprof.py

```diff
@@ -3283,3 +3287,116 @@ def get_cluster_config(self) -> vDataFrame:
         query = """SELECT * FROM v_monitor.host_resources;"""
         query = self._replace_schema_in_query(query)
         return vDataFrame(query)
+
+    def export_profile(self, filename: os.PathLike) -> None:
+        """
+        Creates a tar file containing parquet files of the
+        replica tables used while profiling a query.
+
+        Parameters
+        -----------------
+        filename: os.PathLike
+           The name of the output file to produce. The output file
+           will be a tarball containing parquet files.
+
+        Example
+        -------------------
+
+        ..code-block():: python
+
+            from verticapy.performance.vertica import QueryProfiler
+            # The target_schema argument tells the QueryProfiler
+            # to create a set of replica tables.
+            qprof = QueryProfiler("SELECT date, MONTH(date) AS month,"
```

Comment:
I do not like much the indentation there

Reply:
I replaced this example with a better one with a different format